mixins/eirini/templates/configmap.yaml

```diff
@@ -60,7 +60,8 @@ data:
       server_key_path: "/workspace/jobs/opi/secrets/eirini-server.key"
       tls_port: 8085
       disk_limit_mb: {{ .Values.eirini.opi.disk_limit_mb }}
-      application_service_account: eirini
+      application_service_account: {{ .Values.eirini.opi.staging.application_service_account }}
+      staging_service_account: {{ .Values.eirini.opi.staging.staging_service_account }}
```

Comment:
Wouldn't it make sense to leave the default for `staging_service_account` set to `null` and then use the `default` function to fall back on `application_service_account`:

```
staging_service_account: {{ default .Values.eirini.opi.staging.application_service_account .Values.eirini.opi.staging.staging_service_account }}
```

That way you can overwrite both values with a single setting, if they are still the same name.